auto_show/views.py

```diff
@@ -25,3 +27,7 @@ def get_serializer_class(self, *args, **kwargs):
             return AutoShowSerializerCreate
 
         return self.serializer_class
+
+
+def one():
```

Comment:
Does this functions work. I.e. you've imported `check_balance` task from `tasks.py` but it's not implemented

Reply:
+